src/nvim/ui.c

```diff
@@ -541,10 +535,11 @@ static void flush_cursor_update(void)
   }
 }
 
-// Notify that the current mode has changed. Can be used to change cursor
-// shape, for example.
-static void ui_mode_change(void)
+/// Check if current mode has changed.
+/// May update the shape of the cursor.
+void ui_cursor_shape(void)
```

Comment:
This change doesn't narrow anything, only removes useless static alias in `ui.c`. This is the name actually used around the code base, but that can be changed, of course.